src/3rdparty/walkontable/src/overlay/bottom.js

```diff
@@ -21,20 +21,19 @@ export class BottomOverlay extends Overlay {
     return CLONE_BOTTOM;
   }
 
-  /**
-   * Cached value which holds the previous value of the `fixedRowsBottom` option.
-   * It is used as a comparison value that can be used to detect changes in that value.
-   *
-   * @type {number}
-   */
-  cachedFixedRowsBottom = -1;
-
   /**
    * @param {Walkontable} wotInstance The Walkontable instance.
    */
   constructor(wotInstance) {
     super(wotInstance);
     this.clone = this.makeClone(CLONE_BOTTOM);
+    /**
```

Comment:
I wouldn't move the property from the class filed to the constructor. IMO it's much cleaner to keep props declared in the class fields.

Here I would reassign the value only.

Reply:
Done within: 3b80e9a.